plugins/env/app/views/samson_env/_fields.html.erb

```diff
@@ -1,9 +1,15 @@
 <fieldset>
   <legend>
-    Env variables written to the .env file or added to kubernetes before deploy
-    <%= additional_info SamsonEnv::HELP_TEXT %>
+    Environment Variables
   </legend>
 
+  <% if ENV['DEPLOYMENT_ENV_REPO'] %>
+    <%= form.input :use_env_repo, label: "Use GitHub repository to manage deployment environment variables",
+                   as: :check_box,
+                   help: 'Use GitHub repository (set by DEPLOYMENT_ENV_REPO) to store environment configuration'
```

Comment:
help does not really say anything else, can remove

Reply:
I'll make this change